_config.yml

```diff
@@ -42,31 +33,98 @@ navigation:
   internal: true
 
 
-# Repo list
-# List repos that you would like to appear on the homepage here
-repos:
-- name: regulations-parser
-  description: Parse regulations
-  url: https://github.com/cfpb/regulations-parser
-- name: regulations-core
-  description: Regulations API
-  url: https://github.com/cfpb/regulations-core
-- name: regulations-site
-  description: Display the regulations
-  url: https://github.com/cfpb/regulations-site
-
-support_repos:
-- name: regulations-bootstrap
-  description: Bootstrap eRegulations
-  url: https://github.com/cfpb/regulations-bootstrap
+general_repos:
+  - name: regulations-parser
+    url: https://github.com/18F/regulations-parser
+    short: Parse regulations
+    long: >
+      The parser effectively converts print-centric documents from the Federal
+      Register, Government Printing Office, etc., pulls out semantic meaning
+      from them (e.g. paragraph structures, citations, definitions), and
+      writes those structures to a data store. The parser also contains tools
+      to help developers debug this process.
+  - name: regulations-core
+    url: https://github.com/18F/regulations-core
+    short: Regulations API
+    long: The store for the parsed regulatory data.
+  - name: regulations-site
+    url: https://github.com/18F/regulations-site
```

Comment:
I tried to hand wave away the confusion between forks in these lines:

> There are many individual repositories that make up eRegulations; this is
> complicated by having two primary forks maintained by CFPB and 18F
> respectively. While these forks will ultimately be merged and maintenance
> shared between multiple agencies, those interested in using eRegs now should
> generally prefer the 18F repositories.

To answer your questions, though, 18F's repos are much more applicable to X-arbitrary-agency. I'd be in favor of moving these repos into the `eregs` organization, but had a pressing need for docs. Shall I create an issue for discussion re: moving the repo ownership? What should we do in the short term?